### PR TITLE
Revised tag search

### PR DESCRIPTION
**Pull request by HeadBeeGuy**
This represents the tag selection system that @tkopdx created.

I'm still getting the hang of pull requests, so this might not be the most elegant way to do things. Let's give it a shot and see what happens!

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.file_watcher = ActiveSupport::FileUpdateChecker
```
Comment (HeadBeeGuy) [inline config/environments/development.rb:61]: Just to check, what was the reason to change the file_watcher? To be honest, it's one of the many bits of Rails that I'm unfamiliar with.

Comment (tkopdx) [reply config/environments/development.rb:61]: Previously, I had to manually restart the docker container to see my changes. This was just a fix I found.

Comment (HeadBeeGuy) [reply config/environments/development.rb:61]: Ah, okay! Maybe that'll be useful in the future!

I think I'll make a test branch to verify that nothing goes awry, and then merge it into the main branch!

Comment (tkopdx) [reply config/environments/development.rb:61]: Sounds great!

**Pull request merged**